sql-ledger.css

```diff
@@ -1,570 +1,388 @@
-/* @override http://ledger.cognita.ch/css/sql-ledger.css */
```

Comment:
@NS huge changes, can't see any difference, something like the default css if no user css is selected?

Reply:
@sw ns doesn't know, where is that css used?